cpdb/cpdb.c

```diff
@@ -229,13 +229,14 @@ char *cpdbGetUserConfDir()
     return NULL;
 }
 
-char *cpdbGetSysConfDir()
+char *cpdbGetSysConfDir(bool *is_dynamic)
```

Comment:
API-wise, I think it'd be nicer to just always dynamically allocate the string, i.e. create a copy if necessary.

Reply:
Done